internal/pkg/manifest/lb_web_svc.go

```diff
@@ -23,6 +23,10 @@ const (
 	DefaultHealthCheckGracePeriod = 60
 )
 
+const (
+	GRPCProtocol = "gRPC" // GRPCProtocol is the HTTP protocol version for gRPC.
```

Comment:
We should probably want to be consistent with
```go
	httpProtocolVersions = []string{"GRPC", "HTTP1", "HTTP2"}
```

Reply:
Those values are the ones that are required by CFN, whereas the typical spelling is ["gRPC"](https://grpc.io/). We just ignore the case and move everything to upper case later when transforming the user input, does that make sense?

Reply:
Yeah that makes sense and also why i said this is just very tiny little nit. it's just i think we'd better to be consistent. But yeah if they are all upper case sure that sounds good to me. Maybe the var name should be `httpProtocolVersionsUpperCase`? You decide boss.